apps/dotcms-ui/src/app/portlets/dot-edit-page/content/dot-edit-content.component.scss

```
@@ -113,3 +125,13 @@ dot-whats-changed,
 ::ng-deep .p-unselectable-text .dot-edit__contentlet-iframe {
     pointer-events: none;
 }
+
+dot-content-palette {
+    transition: all 1s ease-in-out;
```

```suggestion
    transition: ??? 1s ease-in-out;
```

What are we animating here?

---

remove